src/line_image_to_text/filter_images_and_create_csv.py

```diff
@@ -0,0 +1,60 @@
+from pathlib import Path
+import csv
+from utils import get_image_name, add_row_to_csv
+
+
+def get_csv_data(batch_id):
```

Comment:
add extension for function args

for other functions as well
def get_csv_data(batch_id:str):
 ......